azkaban-common/src/main/java/azkaban/project/validator/XmlValidatorManager.java

```diff
@@ -238,7 +231,7 @@ private void parseItemTag(final Node node, final Props props) {
 
   @Override
   public ProjectValidator getDefaultValidator() {
-    return this.validators.get(DEFAULT_VALIDATOR_KEY);
+    return null;
```

Comment:
why null?

Reply:
With the new change, there won't be any default validator. Anyway it will return null if you try to get the default validator from validators map. We plan to deprecate getDefaultValidator() method.

Reply:
Is it ok to remove this method?

Reply:
Since this method is overriding the one in ValidatorManager interface, we need to remove both together. Currently I put @Deprecated in azkaban.project.validator.ValidatorManager#getDefaultValidator, but I think it's ok to remove it entirely.